server/src/main/java/com/google/coffeehouse/common/MembershipConstants.java

```diff
@@ -22,6 +22,8 @@
   public static final int OWNER = 2;
 
   public static final String PERSON_ALREADY_IN_CLUB = "Person is already a member of club.";
+  public static final String NO_CLUBS = "No clubs that the person is a \"%s\" of.";
```

Comment:
nit: it doesn't seem like this comment is called in the code; consider removing.

Reply:
Thanks for pointing this out, good catch Jimmy!